README.md

```diff
@@ -1,9 +1,259 @@
-netx.ws
-=======
+# netx.ws
 
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/kaazing/gateway?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status][build-status-image]][build-status]
 
 [build-status-image]: https://travis-ci.org/kaazing/netx.ws.svg?branch=develop
 [build-status]: https://travis-ci.org/kaazing/netx.ws
 
-WebSocket RFC-6455 implementation
+netx.ws is a Java-based implementation of [WebSocket RFC 6455](https://tools.ietf.org/html/rfc6455) client with following
+important characteristics:
+
+* Target JVM Version - Java SE 1.6
+* `java.net.Socket` and `java.net.URLConnection` style APIs
+* API for streaming binary and text payloads across WebSocket frames
+* API for sending and receiving text and binary messages
+* Extension SPI for extension developers
+* Authentication
+* HTTP Redirect Policies
+* Thread-safe
+* Garbage-free
+* Wait-free
+
+> Using the following maven dependency, developers can include netx.ws to develop and run web applications:
+
+```xml
+<dependency>
+    <groupId>org.kaazing</groupId>
+    <artifactId>netx.ws</artifactId>
+    <version>[0.1,1.0)</version>
+</dependency>
+```
+
+## Building this Project
+
+### Minimum requirements for building the project
+
+* Java SE Development Kit (JDK) 7 or higher
```

Comment:
Does it not build with JDK 6 ?
It's okay if that's really true, but might be unexpected if we are targeting Java6 APIs.  Just need to confirm please.